src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,43 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty lines before the class declaration.